meerk40t/core/parameters.py

```diff
@@ -106,6 +109,14 @@ def validate(self):
         for v in COLOR_PARAMETERS:
             if v in settings:
                 settings[v] = Color(settings[v])
+        for v in LIST_PARAMETERS:
+            if v in settings:
+                if isinstance(settings[v], str):
+                    try:
+                        settings[v] = eval(settings[v])
```

Comment:
This tends to be a security issue. I can't validate that your loaded settings are from a local or safe source. Calling eval on something that may come from nefarious sources isn't great.

Reply:
Also, if it's a list you can get a string and usually use a character as a delimiter.